src/main/resources/org/jenkinsci/plugins/workflow/support/steps/input/InputStep/help-submitter.html

```diff
@@ -1,3 +1,3 @@
 <div>
-    User ID or <em>external</em> group name of person or people permitted to respond to the input.
+    User IDs and/or <em>external</em> group names of person or people permitted to respond to the input, splitted by ','.
```

Comment:
“Splitted” is not a word in English. “Split” is the passive participle, but probably you meant “separated” in this case anyway.